platform-mbean/src/main/java/org/jboss/as/platform/mbean/PlatformLoggingMXBeanAttributeHandler.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.jboss.as.platform.mbean;
+
+import java.lang.management.ManagementFactory;
+import java.lang.management.PlatformLoggingMXBean;
+
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.as.controller.descriptions.ModelDescriptionConstants;
+import org.jboss.as.platform.mbean.logging.PlatformMBeanLogger;
+import org.jboss.dmr.ModelNode;
+
+/**
+ * Handles read-attribute  for the resource representing {@link java.lang.management.PlatformLoggingMXBean}.
+ */
+class PlatformLoggingMXBeanAttributeHandler extends AbstractPlatformMBeanAttributeHandler {
+
+    static final PlatformLoggingMXBeanAttributeHandler INSTANCE = new PlatformLoggingMXBeanAttributeHandler();
+
+    private PlatformLoggingMXBeanAttributeHandler() {
+    }
+
+    @Override
+    protected void executeWriteAttribute(OperationContext context, ModelNode operation) throws OperationFailedException {
+        // Shouldn't happen; the global handler should reject
+        throw unknownAttribute(operation);
+
+    }
+
+    @Override
+    protected void executeReadAttribute(OperationContext context, ModelNode operation) throws OperationFailedException {
+
+        final String name = operation.require(ModelDescriptionConstants.NAME).asString();
+
+        try {
+            if ((PlatformMBeanConstants.OBJECT_NAME.getName().equals(name))
+                    || PlatformLoggingResourceDefinition.PLATFORM_LOGGING_READ_ATTRIBUTES.contains(name)) {
+                storeResult(name, context.getResult());
+            } else {
+                // Shouldn't happen; the global handler should reject
+                throw unknownAttribute(operation);
+            }
+        } catch (SecurityException | UnsupportedOperationException e) {
+            throw new OperationFailedException(e.toString());
+        }
+
+    }
+
+    static void storeResult(final String name, final ModelNode store) {
+
+        if (PlatformMBeanConstants.OBJECT_NAME.getName().equals(name)) {
+            store.set(PlatformMBeanConstants.PLATFORM_LOGGING_MXBEAN_NAME);
+        } else if (PlatformMBeanConstants.LOGGER_NAMES.equals(name)) {
```

Comment:
@jamezp is PlatformLoggingMXBean.getLoggerNames a complete list in the WF context, i.e. the log manager is from JBoss Logging?

Reply:
It should be, yes. It should return any logger created with:
* JUL
* JBoss Logging
* JBoss Log Manager
* SLF4J
* Log4j 2.x
* Apache Commons Logging

Reply:
Maybe I didn't answer completely. This should be a list of all loggers created, not just the ones defined in the logging subsystem which contain the configuration for the logger.

Reply:
@jamezp , that is what I am observing. We have the list of all loggers. That would provide a source of information of what are the loggers that could be enabled.

Reply:
Great; thanks!